Add upgrade script to fix missing Pronamic subscription IDs in WooCommerce subscription meta

rvdsteege opened the pull request:
This PR aims to resolve issue #11, by adding an upgrade script which adds the `pronamic_subscription_id` meta to WooCommerce subscriptions not having this meta.

rvdsteege commented:
@remcotolsma would https://github.com/pronamic/wp-pronamic-pay-woocommerce/pull/12/commits/5002eb21833ed17144090947bbb871f1a09b0923#diff-4c7a9bafcdb53c32fa769721fb6cd3ca59ebb28cc2b37d881e0541547fe26c73R116 be an issue for larger sites? Thoughts on how to improve?

rvdsteege commented on src/Upgrade420.php:
```diff
@@ -0,0 +1,307 @@
+	private function get_subscription_posts( $args = [] ) {
+		$args['post_type']     = 'pronamic_pay_subscr';
+		$args['post_status']   = 'any';
+		$args['nopaging']      = true;
```
@remcotolsma would this be an issue for larger sites? Thoughts on how to improve?

remcotolsma replied:
@rvdsteege ideally we should set it up like in https://github.com/pronamic/wp-pay-core/blob/046ed470ff0966419fda8ae663a9630b95b0c70a/src/Subscriptions/SubscriptionsFollowUpPaymentsController.php#L141-L178 ?

remcotolsma commented:
@rvdsteege I reverted https://github.com/pronamic/wp-pay-core/commit/9a75a8dadff2470c0c2517e59751206efd4a1067 in https://github.com/pronamic/wp-pay-core/commit/3512f889c5907ed9d3675d280968c5273c9614fc, because we can't use the class `QueryActionsScheduler` more widely for now, it is a bit of a `YAGNI` case. I tried to marge this back in https://github.com/pronamic/wp-pronamic-pay-woocommerce/commit/4f1f4919889d9040cf1acf32bed22be4ab0c1c04, but have not tested this yet. We also spoke about this code:

https://github.com/pronamic/wp-pronamic-pay-woocommerce/blob/4f1f4919889d9040cf1acf32bed22be4ab0c1c04/src/Extension.php#L86-L92

https://github.com/pronamic/wp-pronamic-pay-woocommerce/blob/4f1f4919889d9040cf1acf32bed22be4ab0c1c04/src/Extension.php#L137-L146

That requires some adjustments?

And this:

https://github.com/pronamic/wp-pronamic-pay-woocommerce/blob/4f1f4919889d9040cf1acf32bed22be4ab0c1c04/src/Upgrade420.php#L160-L164

Delete action ID should be done afterwards?

rvdsteege commented:
> That (upgrades executable) requires some adjustments?

Updated in https://github.com/pronamic/wp-pronamic-pay-woocommerce/pull/12/commits/4a4a68908cf8226644b138576d12ddf69145a77c.

> Delete action ID should be done afterwards?

Updated in https://github.com/pronamic/wp-pronamic-pay-woocommerce/pull/12/commits/ae4a27fb9b87c25da3d7371be763c13fe461c3d3.
